inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -1440,6 +1440,29 @@ void FuseMultipleIdentitiesPass::run() {
     }
 }
 
+void PowerBroadcastPass::run() {
+    for (auto &l : *pLayers) {
+        if (!LayerInfo(l).isPower()) {
+            continue;
+        }
+
+        auto constDims = l->outData.front()->getTensorDesc().getDims();
+        if (constDims.size() == 1) {
```

Comment:
This is weird check. Was your intention to verify if at least 2 dimensions are >1 ?